pkgs/development/compilers/graalvm/community-edition/mkGraal.nix

```diff
@@ -86,7 +81,7 @@ let
       zlib
     ];
 
-    nativeBuildInputs = [ unzip perl autoPatchelfHook makeWrapper ];
+    nativeBuildInputs = [ unzip perl makeWrapper ] ++ lib.optional stdenv.isLinux [ autoPatchelfHook ];
```

Comment:
You can either try to match the exactly change in `staging` (this may or may not work), or rebase your PR based on `staging` (this is safer however it also means your PR will only go to `master` on the next `staging` merge).